frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -30,6 +30,8 @@ import { VMIKind, VMKind } from '../../types/vm';
 import { getLoadedData } from '../../utils';
 import { VMTabProps } from './types';
 
+import './_vm-consoles.scss';
```

Comment:
To stay consistent with rest of the code, we should leverage `kubevirt-plugin/_style.scss`

Reply:
done